server/server/routes.py

```diff
@@ -3,3 +3,5 @@ def includeme(config):
     config.add_route('home', '/')
     config.add_route('log_in', '/log-in')
     config.add_route('log_out', '/log-out')
+    config.add_route('registration', '/registration')
```

Comment:
I believe you need update your branch with latest changes to `dev`, as there is cornice is used. And you will need to update your code to use this framework.

Reply:
I remember it, just I want to close this step and after it add the cornice.